lib/players/human.rb

```diff
@@ -0,0 +1,13 @@
+require_relative '../player.rb'
+
+module Players
+	attr_reader :move
+
+	class Human < Player
+		
+		def move(board, game=Game.new)
+			input = gets.chomp
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "Where would you like to move?"
  gets.chomp
end
```